docs/reference/search/request/script-fields.asciidoc

```diff
@@ -50,7 +50,7 @@ GET /_search
         },
         "script_fields" : {
             "test1" : {
-                "script" : "_source.obj1.obj2"
+                "script" : "params._source.obj1.obj2"
```

Comment:
Ooh! If you add `// TEST[setup:twitter]` then below `// CONSOLE` on this page then we'll actually try an execute the script which might be nice.